config/data.php

```diff
@@ -96,7 +96,7 @@
         'enabled' => true,
         'directories' => [app_path('Data')],
         'cache' => [
-            'store' => env('CACHE_DRIVER', 'file'),
+            'store' => env('CACHE_STORE', 'file'),
```

Comment:
```suggestion
            'store' => env('CACHE_STORE', env('CACHE_DRIVER', 'file')),
```

To be compatible with both versions

Reply:
Sure, fixed